### ISSUE TITLE
Bottom border issue

### ISSUE THREAD

**Issue by MmKolodziej** [gTile/gTile#39]
I haven't found any answer to this nor anyone mentioning a problem like this. Using 2x2 grid from presets on Ubuntu 17.10 with Gnome 3.26 results in borders (especially visible at the bottom) as shown in the following screenshot. It is much more apparent with terminal than other windows (sublime for comparison on the right).
![screenshot from 2018-05-21 11-41-58](https://user-images.githubusercontent.com/5260236/40301325-0f43631c-5cec-11e8-8af9-2e222cfb9497.png)
Am I missing anything in regards to configuration or can do on my side to improve this or is this a legit issue?

**Comment by scherepanov** [comment]
Gnome terminal is a special thingy. After move, it is adjusting it's size to slightly smaller to match font size and char count. It tries to keep it's window to avoid trimming part of characters.
That result in visible space below and on a right of Gnome terminal, if it is adjusted with help of gTile.
From other side, if you will try to use gnome built-in windows movement (Super - left or Super-right) it does not adjust itself.
Gnome terminal show some other strange and very clever behavior on window resize/move. Seems to be person who wrote it know gnome-shell internals pretty well. I was not able to repeat some tricks done by Gnome terminal.
In short, gTile work correctly, but application is resizing it's own window after move. This is out of control of gTile.

**Comment by MmKolodziej** [comment]
Thank you, makes perfect sense for the terminal. There remains the question of other applications and the super thin border that remains. Note the difference between the first screenshot which is Super-right and the second screenshot which is done with gTile grid.

![screenshot from 2018-05-22 16-23-17](https://user-images.githubusercontent.com/5260236/40368615-80191d8e-5ddc-11e8-85f5-0c178ec60ed0.png)
![screenshot from 2018-05-22 16-23-29](https://user-images.githubusercontent.com/5260236/40368618-836ee4be-5ddc-11e8-8e11-b102fbb20ee8.png)

It seems that Gnome uses some special window mode for Super-left/right as this is the upper corner between applications (once again, first is pure gnome, second is gTile).

![screenshot from 2018-05-22 16-25-33](https://user-images.githubusercontent.com/5260236/40368763-ca466132-5ddc-11e8-811c-4136e190de03.png)
![screenshot from 2018-05-22 16-25-57](https://user-images.githubusercontent.com/5260236/40368785-d7790148-5ddc-11e8-8674-b421adabe2f9.png)

I realize that these are really first-world problems and I really like gTile for what it does but shining white borders/corners peeking from behind dark editor make me feel uneasy. If this is something that's too difficult to fix or not important enough for you, feel free to close this issue.

**Comment by scherepanov** [comment]
Well, after tracing window placement, pixels are calculated correctly and not leaving gaps. At same time I do see barely noticeable very thin black line between two adjacent windows.
I am blaming this line on Gnome effects and style of borders drawing. All I can do is to calculate pixels correctly, and it is correct.
